ipaserver/install/ipa_migrate_constants.py

```diff
@@ -777,6 +783,11 @@
         'subtree': ',cn=subids,cn=accounts,$SUFFIX',
         'label': 'Sub IDs',
         'mode': 'all',  # TODO Maybe production only?
```

Comment:
@rcritten - I suspect "subids" should be production only, can you confirm?

Reply:
Yes, that seems like a good choice.

Reply:
> Yes, that seems like a good choice.

Ok I'll make this change then...